data/json/professions.json

```diff
@@ -2275,7 +2275,7 @@
     "type": "profession",
     "ident": "cosplay",
     "name": "Otaku",
-    "description": "Late nights with friends watching anime and eating snacks has prepared you for the premier anime convention in the Northeast.  It just had to be the day of the apocalypse.  At least you were ready in case your costume tore.",
+    "description": "After many late nights with friends watching anime and eating snacks, you decided to make the trip to the premier anime convention in the Northeast.  They just had to schedule it the day of the apocalypse.  At least you were ready in case your costume tore.",
```

Comment:
Again, there isn't really a day. Though arguably the portal storms could have happened on this day? Though I think the convention would have been cancelled first.